vendor/tests/test_processor.py

```diff
@@ -1083,10 +1083,18 @@ def test_get_expiring_cards_success(self):
         site = Site.objects.get(pk=1)
         processor = AuthorizeNetProcessor(site)
 
-        processor.get_customer_id_for_expiring_cards("2024-01")
+        ids = processor.get_customer_id_for_expiring_cards("2024-01")
 
         self.assertTrue(processor.transaction_submitted)
+        self.assertTrue(ids)
 
+    def test_get_expiring_cards_empty(self):
+        site = Site.objects.get(pk=1)
+        processor = AuthorizeNetProcessor(site)
+
+        ids = processor.get_customer_id_for_expiring_cards("2020-01")
+
```

Comment:
same note as on the other pr- maybe we should add a check for `self.assertTrue(processor.transaction_submitted)` to make sure the check was successful and still returned an empty response. but not blocking over it!